docs/src/reference/asciidoc/core/spark.adoc

```diff
@@ -2016,7 +2016,7 @@ To wit, below is a snippet from the https://spark.apache.org/docs/1.5.1/programm
 
 [source, python]
 ----
-$ ./bin/pyspark --driver-class-path=/path/to/elasticsearch-hadoop.jar
+$ ./bin/pyspark --driver-class-path=/path/to/elasticsearch-hadoop-mr-6.2.4.jar
```

Comment:
This locks this page of documentation into this specific version, which we can't do. Can you keep the jar name as `elasticsearch-hadoop`, but just add the `{ver}` macro on the end of it?